src/main/java/org/folio/services/kafka/KafkaProperties.java

```diff
@@ -9,18 +9,20 @@
 
 public final class KafkaProperties {
   private static int port = parseInt(getenv().getOrDefault("KAFKA_PORT", "9092"));
+  private static String host = getenv().getOrDefault("KAFKA_HOST", "localhost");
 
-  public static int changePort(int port) {
+  public static void setHostPort(String host, int port) {
+    KafkaProducerServiceFactory.clear();
     KafkaProperties.port = port;
-    return port;
+    KafkaProperties.host = host;
   }
```

Comment:
Personally, I'd prefer these were set separately. However I understand that changing either of these is a signal to reset the service factory, and it's easier to do that in one place only.

Reply:
ok